Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM openjdk:11.0.1-jdk-slim AS builder
+
+RUN mkdir /code
+WORKDIR /code
+
+ENV GRADLE_OPTS -Dorg.gradle.daemon=false
+
+COPY ./gradle /code/gradle
+COPY ./gradlew /code/
+RUN ./gradlew --version
+
+COPY ./build.gradle ./settings.gradle /code/
+COPY ./src /code/src
+
+RUN ./gradlew unpack
+
+FROM openjdk:11.0.1-jre-slim
+
+MAINTAINER Yatharth Ranjan <yatharth.ranjan@kcl.ac.uk>
+
+LABEL description="RADAR-base App server"
```

Comment:
The plan is to have this as a generic app server and not just for scheduling notifications. For eg, later it can be used to generate the schedule using protocol and the app will just query the next task from the server.

Reply:
Then could we call it `armt-app-server`.